fineract-provider/src/main/java/org/apache/fineract/infrastructure/creditbureau/service/OrganisationCreditBureauWritePlatflormServiceImpl.java

```diff
@@ -76,13 +76,13 @@ public CommandProcessingResult updateCreditBureau(JsonCommand command) {
 
         final long creditbureauID = command.longValueOfParameterNamed("creditBureauId");
 
-        final boolean is_active = command.booleanPrimitiveValueOfParameterNamed("is_active");
+        final boolean isActive = command.booleanPrimitiveValueOfParameterNamed("isActive");
 
         final OrganisationCreditBureau orgcb = organisationCreditBureauRepository.getOne(creditbureauID);
 
-        orgcb.setIsActive(is_active);
+        orgcb.setIsActive(isActive);
 
-        organisationCreditBureauRepository.saveAndFlush(orgcb);
```

Comment:
you do not want to commit to transaction?

Reply:
@nikpawar89 well spotted! I'm resolving this, as it looks like it has been addressed in the current version (the `saveAndFlush` is no longer removed now). @rrpawar96 Please Resolve open discussion items yourself in the future, as it makes it easier and faster for me to review pending pull requests.